demisto_sdk/commands/download/downloader.py

```diff
@@ -389,6 +389,7 @@ def fetch_custom_content(self) -> bool:
                     if (
                         "automation-" in member.name.lower()
                         or "integration-" in member.name.lower()
+                        or "layout-" in member.name.lower()
```

Comment:
Can the `... in` be replaced for `startswith(...)`?
Could be safer in cases like `script-set-values-in-layout-3`, and prevent false-positives